src/devices/System.Device.Model/README.md

```diff
@@ -0,0 +1,111 @@
+# System.Device.Model - attributes for device bindings
+
+**This library is experimental, it may change and it may be moved to a different package in the future. Avoid taking direct dependency on it.**
+
+This library provides a set of attributes which allow to annotate devices.
+
+They can be used for:
+- implementation of Azure Plug & Play
+- providing extra metadata about sensors (we can i.e. generate some parts of the README file or add extra checks)
+
+Model is representing language independent description of the device. The attributes represent a C# mapping from C# types into the model.
+
+## InterfaceAttribute
+
+Every class producing telemetry or exposing some commands should put a telemetry attribute on it
+
+```csharp
+    [Interface("LPS25H - Piezoresistive pressure and thermometer sensor")]
+    public class Lps25h : IDisposable
+```
+
+- if class derives from class annotated with `[Interface(...)]` attribute:
+  - class will inherit all annotations from the base class(es)
+  - if class provides extra telemetry/command/properties it should add another InterfaceAttribute on itself
+  - if class doesn't provide any extra annotations it should not have extra interface
+- display name must be provided to the InterfaceAttribute
+
+## TelemetryAttribute
+
+Every method or property producing telemetry should have `[Telemetry]` attribute on it.
+For properties providing name of the `Telemetry` is optional as it can be deduced from the property name.
+
+Telemetry can be put on:
+- properties
+- methods returning value but not taking any arguments
+- methods returning bool and taking one `out` argument
+  - multiple `out` arguments are currently out of scope but are considered
+
+```csharp
+        [Telemetry]
```

Comment:
remove the extra space, that's not needed, can do everywhere in the samples from this readme